keyboards/aidansmithdotdev/sango/config.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2024 Aidan Smith <aidan@aidansmith.dev>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define EE_HANDS
+
+// Not yet available in `info.json`
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS RGBLIGHT_LIMIT_VAL
+#    define RGB_MATRIX_LED_COUNT RGBLED_NUM
+#    define RGB_MATRIX_SPLIT RGBLED_SPLIT
+#    define SPLIT_TRANSPORT_MIRROR
+#endif
```

Comment:
These can also be moved to info.json, however, `RGB_MATRIX_LED_COUNT` does not need to be explicitly defined; the CLI can infer it from the layout.